s3-source-connector/src/integration-test/java/io/aiven/kafka/connect/s3/source/IntegrationBase.java

```diff
@@ -59,6 +59,8 @@
 public interface IntegrationBase {
 
     String DOCKER_IMAGE_KAFKA = "confluentinc/cp-kafka:7.7.0";
+    String PLUGINS_S_3_SOURCE_CONNECTOR_FOR_APACHE_KAFKA = "plugins/s3-source-connector-for-apache-kafka/";
```

Comment:
Should it really be `S_3`? This even complicates search in the code.

Reply:
Yeah thats a fair point.

Reply:
We will anyways have more reviews on parent pr. I will make sure this is addressed. There are a couple of more vars like these.

Reply:
And then we will end up with a PR with hundreds(if not thousands) of comments?